.eslintrc.json

```diff
@@ -11,7 +11,7 @@
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
       "rules": {
         "@nrwl/nx/enforce-module-boundaries": [
-          "warn",
```

Comment:
@matthew-garrett just undo these changes so the build compiles, will have @mgnsharon look at the errors